readline/PKGBUILD

```diff
@@ -6,6 +6,7 @@ _basever=7.0
 _patchlevel=003 #prepare for some patches
 pkgver=${_basever}.${_patchlevel}
 pkgrel=1
+epoch=1
```

Comment:
Git for Windows does not modify `readline`. Therefore this change should be unnecessary.